roles/gpu_operator_run_gpu-burn/tasks/main.yml

```diff
@@ -18,20 +18,32 @@
   register: gpu_burn_gpu_nodes
   failed_when: gpu_burn_gpu_nodes.stdout == ""
 
+# Store the configmap into a file, for easy reproduction from artifacts
 - name: Create the entrypoint ConfigMap
-  command: oc apply -f "{{ gpu_burn_cm_entrypoint }}"
-
-- name: Delete the src ConfigMap, if it exists
-  command:
-    oc delete cm/gpu-burn-src
-       --ignore-not-found=true
+  shell:
+    oc create configmap gpu-burn-entrypoint
+       --from-file=entrypoint.sh={{ gpu_burn_entrypoint }}
        -n default
+       --dry-run=client
+       -oyaml
+       > {{ artifact_extra_logs_dir }}/000_configmap_gpu-burn_entrypoint.yml
 
 - name: Create the src ConfigMap
   command:
+    oc apply -f {{ artifact_extra_logs_dir }}/000_configmap_gpu-burn_entrypoint.yml
+
+# Store the configmap into a file, for easy reproduction from artifacts
+- name: Create the src ConfigMap
```

Comment:
The title "Create the src ConfigMa"p" appears 3 times

Reply:
ack, I'll push the fix at the end of the current test